### PR TITLE
Localstorage

### PR DESCRIPTION
**Pull request by thomasmueller87**
Added localstorage, so that user is able to view his saved logs when he is starting the app again.

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,17 +1,45 @@
...
+      ...logs,
+      {
+        ...log,
+        id: addId,
+      },
+    ]);
+  }
+
   return (
     <div className='App'>
```
Comment (gratianpermien) [inline client/src/App.jsx]: I think you may let go of the className :)

File: client/src/components/LogCard.jsx [hunk 2]
```diff
@@ -67,8 +49,8 @@ const CardGrid = styled.div`
     grid-template-areas:
```
Comment (gratianpermien) [inline client/src/components/LogCard.jsx]: Uh, nice!

**Review by gratianpermien — APPROVED** [review]
LGTM - except you will have merging conflicts, see below

**Comment by thomasmueller87** [comment]
Thanks Gratian :)

**Pull request merged**